core/src/main/java/jenkins/widgets/HistoryPageFilter.java

```diff
@@ -191,13 +210,13 @@ public int size() {
         return queueItems.size() + runs.size();
     }
 
-    private void sort(List<T> items) {
+    private void sort(List<? extends Object> items) {
```

Comment:
`T` was kind of inaccurate; `Queue.Item` can be included in these items but they are not related to `T` in any way. (Before this change, the code got around this with a cast to `(Collection<? extends T>)` in BuildHistoryWidget.)